### PR TITLE
Matplotlib ideas (initial draft)

### PR DESCRIPTION
**Pull request by story645**
attn: @tacaswell and @efiring

### CODE REVIEW

**Comment by kain88-de** [comment]
@story645 who would be the mentors and how can writers best reach out to matplotlib?

File: 2019/Matplotlib_ideas_list.md [hunk 1]
```diff
@@ -1 +1,31 @@
...
+__email:__ matplotlib@numfocus.org
 
+
+__Project name:__ Matplotlib for Data Scientists
+
+__Description:__ Matplotlib was primarily develped by scientists for scientific visualization, and modeled on scientific computation languages such as Matlab, but one of the fastest growing audiences for Matplotlib is data science practitioners. To address the needs of this audience, we propose developing a conceptual overview (such as the scikit one linked below) oriented at newcomers to scientific visualization. 
+
+The overview would possibly cover topics such as:
+* the visualization pipeline: data->transformation->chart
+* matplotlib's role in the viz pipeline , i.e. that it's a low level library
```
Comment (kain88-de) [inline 2019/Matplotlib_ideas_list.md]: Can you find a link to information about this.  E.g. https://www.youtube.com/watch?v=FytuB8nFHPQ

File: 2019/Matplotlib_ideas_list.md [hunk 2]
```diff
@@ -1 +1,31 @@
...
+* assumptions matplotlib makes about the structure of the data, user, and task
+* the relationship between matplotlib and libaries built on top, such as Pandas and Seaborn
```
Comment (kain88-de) [inline 2019/Matplotlib_ideas_list.md]: can you give information on this topic? Preferably a blog post/ Stack Overflow posts that already exist and aren't very outdated.

**Review by kain88-de — COMMENTED** [review]
Very good idea. I only left some comments where it can be good to give some more background information.

**Comment by story645** [comment]
@kain88-de I'm the mentor and via the email I in the proposal.

**Comment by kain88-de** [comment]
Thanks

**Comment by kain88-de** [comment]
You need at least one other mentor according to the gsod guidelines. Who would that be?

**Pull request merged**